addons/storysource/src/StoryPanel.js

```diff
@@ -57,29 +102,74 @@ export default class StoryPanel extends Component {
     );
   }
 
-  lineRenderer({ rows, stylesheet, useInlineStyles }) {
-    const { location } = this.state;
+  createStoryPart(rows, stylesheet, useInlineStyles, location, kindStory) {
+    const { currentLocation } = this.state;
+    const first = location.startLoc.line - 1;
+    const last = location.endLoc.line;
+
+    const storyRows = rows.slice(first, last);
+    const story = this.createPart(storyRows, stylesheet, useInlineStyles);
+    const storyKey = `${first}-${last}`;
 
-    if (location) {
+    if (StoryPanel.areLocationsEqual(location, currentLocation)) {
+      return (
+        <div key={storyKey} ref={this.setSelectedStoryRef} style={styles.selectedStory}>
+          {story}
+        </div>
+      );
+    }
+
+    return (
+      <div
```

Comment:
why not just use `button` tag?

Reply:
I think it's semantically more like an anchor than a button so maybe `a` ?

Reply:
Just checked and resetting a button to look like a div - is hell...

Reply:
I'd say that making div behave as a button is hell =)

> I think it's semantically more like an anchor than a button so maybe `a`?

Cool idea as soon as you attach a valid `href`. You may want to use `RoutedLink` component from `@storybook/components` to avoid page reload on plain clicks